servlet/pom.xml

```diff
@@ -65,6 +65,13 @@
             <groupId>org.jboss.spec.javax.annotation</groupId>
             <artifactId>jboss-annotations-api_1.2_spec</artifactId>
         </dependency>
+
+        <dependency>
+            <groupId>org.osgi</groupId>
```

Comment:
It's at scope provided, which means it does not show as a transitive dependency.  I can make it provided + optional if you prefer.